### ISSUE TITLE
Video component on iOS cannot be shown twice at the same time in a Carousel

### ISSUE THREAD

**Issue by kevgrig** [expo/expo#7180]
### Environment

```
$ expo diagnostics

  Expo CLI 3.13.1 environment info:
    System:
      OS: Linux 5.4 Fedora 31 (Workstation Edition) 31 (Workstation Edition)
      Shell: 5.0.11 - /bin/bash
    Binaries:
      Node: 12.15.0 - /usr/bin/node
      Yarn: 1.21.1 - /usr/bin/yarn
      npm: 6.13.4 - /usr/bin/npm
    npmPackages:
      expo: ~36.0.0 => 36.0.2 
      react: ~16.9.0 => 16.9.0 
      react-native: https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz => 0.61.4 
    npmGlobalPackages:
      expo-cli: 3.13.1
```

Target: iOS, Android

### Steps to Reproduce

1. Snack: https://snack.expo.io/@git/github.com/kevgrig/testvideo
2. Only one video is playable in the iOS emulator. Both are playable in the Android emulator. Matches physical device behavior (iPhone SE and BlackBerry KEY2, respectively).

### Expected Behavior

Multiple videos can play at the same time.

### Actual Behavior

Only one video is playable in the iOS emulator. Both are playable in the Android emulator. Matches physical device behavior (iPhone SE and BlackBerry KEY2, respectively).

### Reproducible Demo

https://snack.expo.io/@git/github.com/kevgrig/testvideo
https://github.com/kevgrig/testvideo

### Screenshots

![Screenshot_20200224_163057](https://user-images.githubusercontent.com/6432587/75203579-4ddca880-5723-11ea-8635-0d94ac5f9fc0.png)
![Screenshot_20200224_163201](https://user-images.githubusercontent.com/6432587/75203580-4e753f00-5723-11ea-99f3-68b267d26c93.png)

### Reference

https://github.com/ihmpavel/expo-video-player/issues/71

**Comment by kevgrig** [comment]
Added an onError handler and receive:

```
The AVPlayerItem instance has failed with the error code -1100 and domain "NSURLErrorDomain".
- node_modules/expo/build/environment/muteWarnings.fx.js:27:24 in error
* App.js:29:27 in Video.props.onError
- node_modules/expo-av/build/Video.js:139:27 in _nativeOnError
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:307:15 in invokeGuardedCallbackImpl
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:531:36 in invokeGuardedCallback
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:555:30 in invokeGuardedCallbackAndCatchFirstError
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:722:42 in executeDispatch
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:744:20 in executeDispatchesInOrder
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:907:29 in executeDispatchesAndRelease
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:889:12 in forEachAccumulated
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:932:21 in runEventsInBatch
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:1096:19 in runExtractedPluginEventsInBatch
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:2796:6 in batchedUpdates$argument_0
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:18791:14 in batchedUpdates$1
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:2709:30 in batchedUpdates
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:2794:17 in batchedUpdates$argument_0
- node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:2814:26 in receiveEvent
- node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:436:47 in __callFunction
- node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:111:26 in __guard$argument_0
- node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:384:10 in __guard
- node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:110:17 in __guard$argument_0
* [native code]:null in callFunctionReturnFlushedQueue
```

**Comment by IjzerenHein** [comment]
Hi! I've been trying to reproduce this problem but was unable to. I tried the snack and was able to start the video twice without any problems (on simulator and real device). I tried both on SDK 36 and SDK 38. It's possible that this problem has already been resolved or that it perhaps was related to the carousel component.

In any case, let us know whether this problem is still an issue for you and how it can be reproduced, or it can be closed?

Regards
